vmdb/app/models/container_node.rb

```diff
@@ -3,5 +3,6 @@ class ContainerNode < ActiveRecord::Base
   # :name, :uid, :creation_timestamp, :resource_version
   belongs_to :ext_management_system, :foreign_key => "ems_id"
   has_many   :container_groups
+  has_one    :hardware, :dependent => :destroy
```

Comment:
With this addition, I'm concerned that `ContainerNode` should `has_one ComputerSystem` to be more accurate and to start driving everything in that direction.  cc @blomquisg Thoughts?